ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift

```diff
@@ -173,6 +178,43 @@ class RootViewController: UIViewController, UITableViewDataSource, UITableViewDe
         return tableView
     }()
 
+    private(set) lazy var searchProgressSpinnerSwitchView: UIView = {
+        let itemRow = UIStackView()
+        itemRow.axis = .horizontal
+        itemRow.distribution = .equalCentering
+        itemRow.alignment = .leading
+        itemRow.isLayoutMarginsRelativeArrangement = true
+        itemRow.layoutMargins = UIEdgeInsets(top: 10, left: 10, bottom: 10, right: 10)
+        itemRow.translatesAutoresizingMaskIntoConstraints = false
+
+        let label = Label(style: .subhead, colorStyle: .regular)
```

Comment:
nit: can we name this searchSpinnerSwitchLabel?

Reply:
Comment Incorporated

---
In reply to: [470762551](https://github.com/microsoft/fluentui-apple/pull/174#discussion_r470762551) [](ancestors = 470762551)